onnxmltools/convert/common/_container.py

```diff
@@ -128,13 +130,18 @@ def __init__(self, targeted_onnx):
         self.nodes = []
         # ONNX operators' domain-version pair set. They will be added into opset_import field in the final ONNX model.
         self.node_domain_version_pair_sets = set()
+        # Precision of float numbers: 16 or 32
```

Comment:
Please throw an error if it's neither 16 or 32. Also, do we need to support Double?

Reply:
done